vcf2fhir/test/test_integration.py

```diff
@@ -39,6 +39,173 @@ def _validate_phase_rel(fhir_json, map_variant_index):
+        # Also set the uids to '' to avoid guid comparison in next step
+        map_ids = _get_uids_map(actual_fhir_json)
+        self.assertEqual(map_ids['obv_ids'], map_ids['result_ids'])
+        actual_fhir_json['issued'] = ''
+        # Finally, check if the acutal json after removing all uids is
+        # same as expected json
+        expected_fhir_json = json.load(expected_output_file)
+        self.assertEqual(actual_fhir_json, expected_fhir_json)
+
+
+def _test_empty_and_region_studied(
+        self,
+        output_filename,
+        vcf_filename,
+        region_studied_filename=None,
+        conv_region_filename=None,
+        conv_region_dict=None,
+        nocall_filename=None,
+        has_tabix=False,
+        patient_id=None,
+        build=None):
+    region_studied_filename = (os.path.join(
+        os.path.dirname(__file__),
+        region_studied_filename)
+    ) if region_studied_filename is not None else None
+    conv_region_filename = (os.path.join(
+        os.path.dirname(__file__),
+        conv_region_filename)
+    ) if conv_region_filename is not None else None
+    output_filename = os.path.join(os.path.dirname(
+        __file__), self.TEST_RESULT_DIR, output_filename)
+    o_vcf_2_fhir = vcf2fhir.Converter(
+        os.path.join(
+            os.path.dirname(__file__),
+            vcf_filename),
+        build,
+        patient_id,
+        has_tabix=has_tabix,
+        conv_region_filename=conv_region_filename,
+        conv_region_dict=conv_region_dict,
+        region_studied_filename=region_studied_filename,
+        nocall_filename=nocall_filename)
+    o_vcf_2_fhir.convert(output_filename)
+
+
+def _compare_type_of_converter(
+        self,
+        vcf_filename,
+        region_studied_filename=None,
+        conv_region_filename=None,
+        conv_region_dict=None,
+        nocall_filename=None,
+        has_tabix=False,
+        patient_id=None,
+        build=None):
+    region_studied_filename = (os.path.join(
+        os.path.dirname(__file__),
+        region_studied_filename)
+    ) if region_studied_filename is not None else None
+    conv_region_filename = (os.path.join(
+        os.path.dirname(__file__),
+        conv_region_filename)
+    ) if conv_region_filename is not None else None
+    nocall_filename = (os.path.join(os.path.dirname(
+        __file__), nocall_filename)
+    ) if nocall_filename is not None else None
+    o_vcf_2_fhir = vcf2fhir.Converter(
+        os.path.join(
+            os.path.dirname(__file__),
+            vcf_filename),
+        build,
+        patient_id,
+        has_tabix=has_tabix,
+        conv_region_filename=conv_region_filename,
+        conv_region_dict=conv_region_dict,
+        region_studied_filename=region_studied_filename,
+        nocall_filename=nocall_filename)
+    self.assertEqual(type(o_vcf_2_fhir), vcf2fhir.Converter)
+
+
+def _compare_exception_string(
+        self,
```

Comment:
Please revert this, if you start looking at actual tests you don't understand what you are testing.